pki.spec

```diff
@@ -999,6 +999,9 @@ EOF
     ln -sf /usr/share/java/jboss-annotations-1.2-api/jboss-annotations-api_1.2_spec.jar %{buildroot}%{_datadir}/pki/lib/jboss-annotations-api_1.2_spec.jar
 %endif
 
+# Install
+cp %{_builddir}/pki-%{version}%{?_phase}/base/util/PKICertImport.bash %{buildroot}%{_bindir}/PKICertImport
```

Comment:
@ladycfu / @edewata Is this fine? This is a stand alone shell script; we don't really need to build it, so I've omitted it from CMake and added it directly to the `.spec` file as a file to install.

Reply:
I'm not sure how the packaging for other distributions (e.g. Debian) is done, but potentially they'll have to add a similar code to theirs. If we add the code in CMake they probably will not need to add anything.

Reply:
A fair point. I think I figured out the `install` syntax on CMake. I'll have to add that as improvements to `jss`.